README.md

```diff
@@ -74,37 +101,34 @@ git@github.com:fauziyahzeeyah/port-folio-mobile.git
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## ⭐️ Show your support <a name="support"></a>
+## 🔭 Future Features <a name="future-features"></a>
+
 
-If you like this project please give it a star.
+- [ ] **[Icon befor title]**
+- [ ] **[Profile picture of myself beside name]**
+- [ ] **[webpage footer]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank @GraceHtet and @TamaraDimov
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/fauziyahzeeyah/port-folio-mobile/issues)).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## ❓ FAQ <a name="faq"></a>
 
-- **[Question_1]**
-  Why did I add MIT.md?
-  - [Answer_1]
-    I added MIT license because it gives users express permission to reuse code for any purpose.
-- **[Question_2]**
-  Why did I ignore node_modules/?
-  - [Answer_2]
-    If folder node_modules: exists then npm install will use those vendored libraries and will rebuild any binary dependencies with npm rebuild.
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star and follow me
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT]([./LICENSE](https://github.com/fauziyahzeeyah/port-folio-mobile/commit/e1c33c8b9a7cb94417172f9e617cfd2e06f31fa7)) licensed.
```

Comment:
- [ ] Please, consider that the link is still broken. The correct link to your license file is `https://github.com/fauziyahzeeyah/port-folio-mobile/blob/accessibility-feature/LICENSE`.

> The broken link screenshot:

<img width="1318" alt="Screen Shot 2022-12-23 at 17 13 50" src="https://user-images.githubusercontent.com/49064106/209342020-ff26fe76-2bca-4a17-9945-6d08764969eb.png">